src/wagtail_vector_index/index/model.py

```diff
@@ -38,7 +38,9 @@ def get_documents(self) -> Iterable[Document]:
         querysets = self._get_querysets()
 
         for queryset in querysets:
-            for instance in queryset.prefetch_related("embeddings"):
+            for instance in queryset.prefetch_related(
+                "embeddings", "embeddings__content_type"
```

Comment:
**Nitpick:** I think only having `embeddings__content_type` does the same thing as having both. I do like being more explicit though. I don't quite ever know where I stand on this one.